scholarly_articles/models.py

```diff
@@ -0,0 +1,54 @@
+from django.db import models
+from django.utils.translation import gettext as _
+
+from wagtail.admin.edit_handlers import FieldPanel
+
+from . import choices
+
+
+class ScholarlyArticles(models.Model):
+    doi = models.CharField("DOI", max_length=255)
```

Comment:
Todos os campos são realmente obrigatórios?

Reply:
@Rossi-Luciano quando eu for aprovar irei remover, vou colocar na lista de pastas para ser ignorada!

Reply:
@Rossi-Luciano concordo com a escolha do campo JSONField! Mas somente é importante notar que esse é um campo específico do **Postgres** a partir da versão 9.2.